package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "moleculer-healthcheck-mongo-middleware",
+  "version": "0.0.0",
+  "description": "Wrapper of moleculer-healthcheck-middleware to check liveness of Mongo",
+  "main": "index.js",
+  "scripts": {
+    "pretest": "npm install",
+    "test": "ava",
+    "pretest:debug": "npm run pretest",
+    "test:debug": "ava debug --host 0.0.0.0 --port 9229"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "ava": "^3.5.0",
+    "moleculer-repl": "^0.6.3",
+    "nodemon": "^2.0.2",
+    "@r2d2bzh/moleculer-healthcheck-middleware": "https://github.com/vivien-lelouette/moleculer-healthcheck-middleware"
```

Comment:
ok, we must not forget to change this as soon as the healthcheck middleware is available on the public npm.

Reply:
Done in 7aaff8f0bf1efd712d32b1a83bed58d160778f99